README.md

```diff
@@ -39,6 +39,18 @@ For more details and requirements see [Stolon Architecture and Requirements](doc
 
 [Documentation Index](doc/README.md)
 
+## Installation
+
+For Mac and Linux we can use [brew](https://brew.sh/),
```

Comment:
We never added external packages since they are managed by other people or distros and not related to the project. Writing this in the Readme will look like they are official packages. I'll prefer to put this in a dedicated file in the doc dir (called unofficial_packages.md) and just a link in the readme that states they are unofficial and not supported by the project.

Reply:
Cool @sgotti I will do the changes and amend to this PR.

Reply:
I have made the changes. Please have a look @sgotti